applications/NXxps.nxdl.xml

```diff
@@ -147,7 +147,7 @@
                     </doc>
                 </field>
                 <group name="transformations" type="NXtransformations" recommended="true">
-                    <field name="beam_direction" type="NX_NUMBER" units="NX_TRANSFORMATION">
+                    <field name="beam_direction" type="NX_NUMBER" units="NX_UNITLESS">
```

Comment:
[units="NX_UNITLESS" shoud be units="NX_TRANSFORMATION", see also here the NIAC version](https://github.com/nexusformat/definitions/blame/8b111da0e5db9ed5ffe02cce5a7483420fc19c2c/applications/NXxps.nxdl.xml#L150)